doc/articles/getting-started/wizard/includes/extensions.md

```diff
@@ -42,6 +42,36 @@ Refer to the [documentation](xref:Uno.Extensions.Http.Overview#Refit) to learn h
 dotnet new unoapp -http
 ```
 
+You have three levels of HTTP support to choose from:
+
+- **Basic**
+
+Just wires up `UseHttp()` so you can register your own `HttpClient`‐backed endpoints and get platform-native handlers, cookie clearing, diagnostics, etc.
+
+```bash
+dotnet new unoapp -http basic
+```
+
+- **Refit**
+
+ Includes the Refit extension so you can create a strongly-typed REST Client for an API and register it via `.AddRefitClient<TInterface>`.
+
+ ```bash
+  dotnet new unoapp -http refit
+ ```
+
+ Learn more in the [Refit quick-start](xref:Uno.Extensions.Http.HowToRefit) guide.
+
+- **Kiota**
+
+ Includes the Kiota extension so you can generate a C# client from an OpenAPI/Swagger spec and wire it up with `.AddKiotaClient<TClient>`.
+
+ ```bash
+ dotnet new unoapp -http kiota
+ ```
+
+See [How-To: Create and register a Kiota client for an API](xref:Uno.Extensions.Http.HowToKiota) for details.
```

Comment:
We should align the wording for both Refit and Kiota lines. So either both "Learn more in the X" or "See X for details